components/channel_layout/channel_identifier_router/index.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {withRouter} from 'react-router-dom';
+
+import ChannelIdentifierRouter from './channel_identifier_router.jsx';
+
+function mapStateToProps() {
+    return {};
+}
+
+export default withRouter(connect(mapStateToProps)(ChannelIdentifierRouter));
```

Comment:
Is there a reason this is needed even though `mapStateToProps` is just returning an empty object?

Reply:
I just moved it to the form then gave up and made a separate ticket. Thought I would just save whoever get's that ticket the trouble.

Reply:
Maybe it'd work without the connect at all? Not sure since I haven't used withRouter before